stories/tooltip/index.jsx

```diff
@@ -8,28 +8,39 @@ import { WrappedComponent } from '../helpers/wrapped';
 export default decorator('Tooltip', TooltipDocs, README).add('Tooltip component', () => {
   return (
     <WrappedComponent>
-      <Row style={{ justifyContent: 'space-around' }}>
-        <Col s="2" m="4" l="3">
-          <Tooltip content="Tooltip message will show up here">TOP</Tooltip>
-        </Col>
-        <Col s="2" m="4" l="3">
-          <Tooltip content="Tooltip message will show up here." position="right">
+      <Row style={ {
+        justifyContent: 'space-around',
+        marginBottom: '1.5rem',
+        marginTop: '1rem',
+      } }>
+        <Col s='6' m='6' l='6'>
+          <Tooltip arrow content='Tooltip message will show up here.'
+                   position='right'>
             RIGHT
           </Tooltip>
         </Col>
+        <Col s='6' m='6' l='6' style={{marginLeft: '1rem'}}>
```

Comment:
wrong breakpoints (I know that you just formatted it, but can you make it right here?)
correct format: `xs="6" sm="6" lg="6"`